driver/normalizer/normalizer.go

```diff
@@ -80,17 +80,32 @@ var Normalizers []Mapping = []Mapping{
 
 	// iflipflop / eflipflop, have selector but not names
 	MapSemantic("flip_1", uast.Identifier{}, MapObj(
-		Obj{uast.KeyToken: Var("ident")},
+		Obj{
+			uast.KeyToken: Var("ident"),
+			// could not find any case where this wasn't nil, but this will
+			// detect if we find any
+			"base": Is(nil),
+			"qnames": Any(), // same as ident
```

Comment:
Then:
```suggestion
			"qnames": Var("ident"),
```
Same below.

Reply:
Same but with slightly different format so using the same Var fail the validation.

Reply:
Ah, okay, then it's fine. Just a short comment regarding the difference will be nice.

Reply:
That makes sense, yes.